### ISSUE TITLE
UnicodeDecodeError: 'ascii' codec can't decode byte 0xf8 in position 2: ordinal not in range(128)

### ISSUE THREAD

**Issue by pschonmann** [mapillary/mapillary_tools#348]
### Basic information
* Release version: `0.5.0` 
* System: `Windows`
* Capture Device: `Xiaomi YI`

### Steps to reproduce behavior

 1. d:\openstreetmap\mapillary_tools>mapillary_tools.exe process --verbose --advanced --import_path c:\TEMP\ZPRACE\ --user_name "pschonmann" --geotag_source "gpx" --geotag_source_path "D:\DOWNLOAD\2019-06-07_16-49-56.gpx" --summarize --interpolate_directions --move_duplicates --local_time --rerun

### Expected behavior
Finish the process without error
...

### Actual behavior

Running user_process for 1676 images, skipping 0 images.
Sub process ended
Running import_meta_data_process for 1676 images, skipping 0 images.
Sub process ended
Running geotag_process for 1676 images, skipping 0 images.
Traceback (most recent call last):
  File "mapillary_tools", line 76, in <module>
  File "mapillary_tools\commands\process.py", line 145, in run
  File "mapillary_tools\process_geotag_properties.py", line 89, in process_geotag_properties
  File "mapillary_tools\processing.py", line 278, in geotag_from_gps_trace
  File "site-packages\dateutil\tz\_common.py", line 22, in adjust_encoding
UnicodeDecodeError: 'ascii' codec can't decode byte 0xf8 in position 2: ordinal not in range(128)

**Comment by JackNUMBER** [comment]
Same issue with 
```
mapillary_tools process --advanced --import_path "test" --user_name jacknumber --local_time --geotag_source "gpx" --geotag_source_path "2019-06-11_01-10_Tue.gpx" --overwrite_EXIF_gps_tag
```

**Comment by kouki-T** [comment]
Hi,all. I had the same trouble with mapillary_tools.exe. This problem is caused Python can't read gpx data stored by UTF-8 because the default character code of Python is ASCII. But, even if we change defolt characrter code for UTF-8 from ASCII, **it doesn't work**. 

### ***I use "cp932" character code, it can works and finish all process.***

However, since used Python is packaged in the exe version, we can't change character code.

So, I recommend to use mapillary_tools by pip. 
https://github.com/mapillary/mapillary_tools#custom-installation

Pip version use local installed Python. so we can change Python's default character code.
Download sitecustomize.py from here and add to "python27\lib\site-packages"
https://ux.getuploader.com/Kouki_T/download/1

After adding, run cmd, and Python with interactive mode. 
check this code.
`>>>import sys`
`>>>sys.getdefaultencoding()`
`'cp932'`

I wrote blog about this problem and solution (Sorry, Japanese).
Please check it, https://qiita.com/puriketu99/items/55e04332881d7b679b00

**Comment by ptpt** [comment]
It should be fixed if you update to 0.6.0 (Python3) which handles UTF-8 by default. Please reopen if it doesn't.
